src/hivpy/experiment.py

```diff
@@ -1,34 +1,59 @@
 from dataclasses import dataclass
-
-from .population import Population
+from datetime import date, timedelta
 from .simulation import run_simulation, SimulationConfig
+import logging
+import os
 
+LEVELS = {
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL
+}
 
 @dataclass
-class ExperimentConfig:
-    """Configuration parameters covering all stages of an experiment."""
-    population_size: int
-    simulation: SimulationConfig
+class OutputConfig:
+    output_dir: str
+    logfile: str
+    loglevel: str
+
+    def start_logging(self):
+        logging.basicConfig(filename=self.logfile, level=LEVELS[self.loglevel])
+        logging.info("starting experiment")
+
+
+
+def create_experiment(experiment_param):
+    start_date = date(1980,1,1)
+    end_date = date(2040, 12, 31)
+    interval = timedelta(days=30)
+    population_size = 1000
```

Comment:
If we're overwriting these anyway, we can remove this?

Reply:
I remember why I put them there: it is to ensure some default values in case none have been provided in the config file. But, yes, there should be a different way of doing this

Reply:
Right, I see. If we want that, then the exception block should also catch `KeyError` (to deal with values not specified in the config file)